packages/entities/entities-plugins/src/components/PluginList.vue

```diff
@@ -457,16 +457,16 @@ const errorMessage = ref<TableErrorMessage>(null)
 // transform the entity row to "Applied To" badges
 const aggregateAppliedTo = (row: EntityRow): ({ type: ViewRouteType | null, badgeText: string })[] => {
   const badges = [] as ({ type: ViewRouteType | null, badgeText: string })[]
-  if (row.route?.id) {
+  if (row.route?.id || row.route_id) {
```

Comment:
nit: add some comments as to why this change is needed

Reply:
done